### ISSUE TITLE
"Completed date" and "Completed by" fields showing 'invalid date' and 'ghost'

### ISSUE THREAD

**Issue by sives** [Pliohub/Plio#1312]
We need to remove these fields from the initially pre-loaded card, to avoid displaying "invalid date" and "ghost".  This applies to any card that contains either of these fields.

**Comment by Praxie** [comment]
![screen shot 2018-08-17 at 6 55 18 pm](https://user-images.githubusercontent.com/2095940/44276856-80b20080-a251-11e8-9a5e-e9c8a5dd36d2.png)
@Serhiy91

**Comment by Praxie** [comment]
@Serhiy91 The solution would be to remove Completed date & Completed by fields from the initially preloaded card.

**Comment by Serhiy91** [comment]
https://github.com/Pliohub/Plio/pull/1314

**Comment by Serhiy91** [comment]
@Praxie No, I did not change any subscriptions. I added checkers https://github.com/Pliohub/Plio/pull/1314/files

**Comment by Praxie** [comment]
Looks good.

**Comment by Serhiy91** [comment]
via Hubstaff
User: Serhiy Zinchuk

Project: Plio Connected - https://app.hubstaff.com/projects/329669
Date Range: 08/20/18 - 08/20/18
Work session total: 3:30:00

Grand total: 3:30:00
